Vagrantfile

```diff
@@ -106,6 +106,7 @@ Vagrant.configure("2") do |config|
       # Only execute once the Ansible provisioner,
       # when all the machines are up and ready.
       if i == $num_instances
+        config.vm.provision :shell, inline: '[[ -x "$(command -v yum)" ]] && ifup eth1', privileged: true
```

Comment:
Did you try the 'centos/7' box?  This one specifically gives us issues.  Using vagrant 1.9.1 and virtualbox 5.0.